Documentation/supermuc.rst

```diff
@@ -196,3 +125,170 @@ Many post-processing scripts of SeisSol require Python dependencies.
 We describe how to use pip on SuperMUC at see :ref:`pypi_behind_firewall`.
 
 
+Using Sanitizer
+---------------
+
+Note that to use sanitizer (https://en.wikipedia.org/wiki/AddressSanitizer), SeisSol needs to be compiled with gcc.
+For that modules and compiler need to be switched:
+
+.. code-block:: bash
+
+    module switch seissol-env seissol-env/develop-gcc11
+
+    export CC=mpigcc
+    export CXX=mpigxx
+    export FC=mpifc
```

Comment:
`FC` is not needed anymore—also `gcc` and `gxx` should be enough actually—CMake will do the rest automatically.